### ISSUE TITLE
Programmatically create or  Update  a Superset role

### ISSUE THREAD

**Issue by codegeekjk** [apache/superset#14659]
Currently no docs exist for creating a role programmatically - it's only via the UI.  Is there any way via which we can create a new role or update the existing role programmatically ?

**Comment by coolsam7486** [comment]
Yes, Its possible actually, You can check test def create_user_with_roles in base_test.py

**Comment by fxulusoy** [comment]
I am also looking for creating a custom role. Is it possible to have this feature on the superset CLI?

**Comment by coolsam7486** [comment]
> It doesn't seem there is a way to do this through the API currently. I'm looking for other methods that might serve this purpose.

You can create one as per your requirement and expose it in superset_config

**Comment by fxulusoy** [comment]
@coolsam7486 Can you give a bit more details how to do it in superset_config?
